frontend/src/app/testResults/operations.graphql

```diff
@@ -0,0 +1,18 @@
+query getTestResultForResendingEmails($id: ID!) {
+  testResult(id: $id) {
+    dateTested
+    patient {
+      firstName
+      middleName
+      lastName
+      email
+    }
+    patientLink{
+      internalId
+    }
+  }
+}
+
+mutation resendTestResultsEmail($patientLinkId: ID!) {
+  sendPatientLinkEmail(internalId: $patientLinkId)
+}
```

Comment:
nitpick: newline

Reply:
We should have more `.graphql` files instead of shoving the query and mutation definitions in the React component modules 👍